src/librbd/internal.cc

```diff
@@ -2221,28 +2221,6 @@ int validate_pool(IoCtx &io_ctx, CephContext *cct) {
     return 0;
   }
 
-  int flush(ImageCtx *ictx)
-  {
-    CephContext *cct = ictx->cct;
-    ldout(cct, 20) << "flush " << ictx << dendl;
-
-    int r = ictx->state->refresh_if_required();
-    if (r < 0) {
-      return r;
-    }
-
-    ictx->user_flushed();
-    C_SaferCond ctx;
-    {
-      RWLock::RLocker owner_locker(ictx->owner_lock);
-      ictx->flush(&ctx);
-    }
-    r = ctx.wait();
-
-    ictx->perfcounter->inc(l_librbd_flush);
```

Comment:
@dillaman Are you going to leave "flush" counter unused?

Reply:
BTW, in "qa/workunits/rbd/test_admin_socket.sh" we have a test that checks "flush" perfcounter is updated after "flush" admin socket command.

Reply:
Yeah -- I thought it was already covered here [1] but it's for AIO flush. I'll combine them in the next push.

[1] https://github.com/ceph/ceph/blob/9b18093892b093daf09a3315e41da2a7f58ac351/src/librbd/io/ImageRequest.cc#L753

Reply:
... consolidated